cavity_code

```diff
@@ -0,0 +1,507 @@
+!==========================2D CAVITY FLOW=========================================  
```

Comment:
Could you create a new direcotry `code/cavity2d/` and move the file there?  And please remove the file `code/placeholder` when you move the file.